### PR TITLE
Fixes Issue #3: Added TestPlan.md

### LINKED ISSUE #3: Generate a test plan for TDD
Write an initial test plan following the general goals of this project. The test plan can be used as guideline on how we'll develop the project. It'll currently be largely generalized and we'll add to it as the project grows. It should cover:

* **Overall description of the project and our goals.**
  This will keep us on track with an objective in mind as we build the project
* **Scope** 
   What is the scope of our minimal viable project (MVP) and the testing that needs to be done at each stage.
* **General architecture of the project.** 
   The different parts that needs to be done and tested for the MVP. Describe system and integration tests that we should think about.

### PR DESCRIPTION
**Pull request by jeanhl**
Added a basic TestPlan.md for the beginnings of a test plan for Some User Experience. TestPlan.md covers our current goals of SUX, a general architecture of the project, testing strategies and future steps to keep in mind as we progress. 

Note: Accidentally named the branch issue_2. Should be issue_3

Closes #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+## Authors
+
+* **Matt Brandt** - *Initial work* - [App Name](https://github.com/m8ttyB/sux)
+* **[Kimbery Pennington](https://github.com/kimberlythegeek)
+* **[Jean Liu](https://github.com/jeanhl)
+
+See also the list of [contributors](https://github.com/your/project/contributors) who participated in this project.
+
+## License
```
Comment (m8ttyB) [inline README.md:67]: You can directly add the license to the readme and delete the license doc. Use this string

`This Source Code Form is subject to the terms of the Mozilla Public
License, v. 2.0. If a copy of the MPL was not distributed with this
file, You can obtain one at http://mozilla.org/MPL/2.0/.`

File: TestPlan.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+#Test plan for Some User Experience (SUX)
+
+## TABLE OF CONTENTS</br>
+Introduction</br>
```
Comment (m8ttyB) [inline TestPlan.md:4]: Instead of line breaks, use markdown syntax.

`###Scope`
`##Testing Strategy`
`* Unit Testing`
`* System and Integration Testing`

**Review by m8ttyB — CHANGES_REQUESTED** [review]
A few small nits and then I think we can merge this.

**Comment by kimberlythegeek** [comment]
Hi @jeanhl, I apologize for the delay! 

Thank you for your contributions! You've clearly spent a lot of time on this.

**Pull request merged**